ui/apps/platform/src/Containers/ConfigManagement/Dashboard/widgets/UsersWithMostClusterAdminRoles.js

```diff
@@ -75,7 +75,12 @@ const UsersWithMostClusterAdminRoles = ({ match, location }) => {
     }
 
     return (
-        <Query query={QUERY}>
+        <Query
+            query={QUERY}
+            variables={{
+                query: 'Cluster Role:true',
```

Comment:
This will not work yet. If you open the `Users and Groups` and focus on the search bar, you will notice that this is not in the search options for subjects. 
For this to work, the search options for the search category must be expanded to include `Cluster Role`.

Reply:
I think this will work only for the 'Users with most cluster admin roles' widget as the query is passed as a string const and not using the result of search options to extract the `Cluster Role` field. But I agree, this would not work on `Users and Groups` page because that will show available filters based on the response of searchOptions. I will create  a separate PR to add the search option while this one can still be merged.